src/main/java/com/chariotsolutions/meetupnotifier/AppBean.java

```diff
@@ -0,0 +1,5 @@
+package com.chariotsolutions.meetupnotifier;
+
+public interface AppBean {
```

Comment:
AppBean, etc. is left over from when I was working with Ken. I just took the files we started and used them in the project, without changing the name to something more meaningful.